packages/logger/src/index.ts

```diff
@@ -1,32 +1,42 @@
-import * as debug from 'debug';
-import * as winston from 'winston';
+import debug from 'debug';
+import { format, Logger, createLogger, transports } from 'winston';
+import { TransformableInfo } from 'logform';
+import stringify from 'json-stringify-safe';
+import { EOL } from 'os';
 
 const { LOG_LEVEL = 'debug' } = process.env;
 
-const formatLogLevel = (level: string): string => {
-  // Clean [31m and invisible character used to show colored strings by Winston
-  const cleaned = level.replace(/\[\d{2}m{1}|[^a-zA-Z ]/g, '');
-  switch (cleaned) {
-    case 'info':
-      return `🍺 ${level}`;
-    case 'warn':
-      return `❗️ ${level}`;
-    case 'error':
-      return `🔥 ${level}`;
-    default:
-      return `🤷‍♂️ ${level}`;
-  }
+const LevelEmoji: Record<string, string> = {
+  info: '🍺',
+  warn: '❗️',
+  error: '🔥',
+  default: '🤷‍♂️',
 };
 
-const instance: winston.Logger = winston.createLogger({
+// have to cast to any to access additional params value
+const splatIndex = Symbol.for('splat') as any;
+
+const emojiLevel = format(
```

Comment:
I would rename this function to something that gives a better indication of what it returns. The current name would indicate it just returns the level emoji, which it also does, but it returns the entire stringified parameters too. I would suggest creating the following:

```
getEmojiLevel = ({ level }: TransformableInfo) => LEVEL_EMOJI[level] || LEVEL_EMOJI.default;
```

Combining this with the comment above to add another function, I would just call it `customFormat(...)` or something like that as a function name 🤷‍♂️